site/content/contribute/more-info/server/schema-migration-guide.md

```diff
@@ -44,16 +339,9 @@ The problem arises when in some databases, for some tables, due to various techn
     - If it's applied you can revert it using morph CLI command. An example command would look like `morph apply down --driver {your-driver} --dsn "{your-dsn}" --path {path-to-your-driver-specific-migration-files} --number 1`
 2. If the migration has been shipped in a release and you want to apply fixes, instead of changing the existing script, you should add a new one so that `db_migrations` will stay consistent. You can edit the existing migration to be a no-op for future releases in this case.
 
-### How do I measure the impact of the migration?
```

Comment:
I think we should still have some sort of imapct analysis of migration (for the obvious ones) 0/5

Reply:
The whole idea is actually covered by the earlier sections where we talk about how we avoid non-blocking migrations. So we don't really need to extract the raw SQL anymore. I am trying to keep things simple. But perhaps, we can keep a line on calculating schema migration time. I'll add that as a last step in the previous section.

Reply:
Yep, a single sentence would be great. We may still want some review/discussion mechanism on any migrations that would happening on large tables.

Reply:
Yep the line is added now. And yes, we have CODEOWNERS in the migrations directory now. So all and any migration will have to be reviewed by someone from our team. We will discuss any large migration on the PR itself.